libs/application/templates/directorate-of-immigration/citizenship/src/fields/Review/DocumentReview.tsx

```
@@ -91,64 +91,70 @@ export const DocumentReview: FC<Props> = ({
+                        return FileList
+                      })}
+                  </GridColumn>
+                )
+              }
 
-                      return FileList
-                    })}
-                </GridColumn>
-              )
+              return null
             })}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)